docs/in_production.rst

```diff
@@ -64,4 +67,32 @@ Configuring realtime data ingestion
 Monitoring Pinot
 ~~~~~~~~~~~~~~~~
 
+In order for Pinot to provide effective service there is a core set of metrics which should be monitored to ensure service stability, fault tolerance and acceptable response times. In the section following, there are service level metrics which are recommended to be monitored.
+
+More info on metrics collection and viewing a complete set of available metric is available in the `Metrics <customizations.html#metrics>`_ section.
+
+Pinot Server
+
+* Missing Segments - Number of missing segments - `NUM_MISSING_SEGMENTS <https://github.com/apache/incubator-pinot/blob/master/pinot-common/src/main/java/org/apache/pinot/common/metrics/ServerMeter.java>`_
```

Comment:
Consider format
```
* <Metric Name/Title>: <code link>

    * <Discription of Metric>
    * <Things to look for/what patterns might mean>
```

Reply:
Probably good to give units as well where applicable (for example for consumption status, boolean might be confusing)

Reply:
Also consider ordering metrics by importance.